src/index.ts

```diff
@@ -9,15 +9,32 @@ export interface Configuration {
 }
 
 export interface HtmlFileConfiguration {
+    /** @param filename The name of the output HTML file (relative to the output directory) */
```

Comment:
Praise: thanks for adding these here!

Reply:
I love having these autocomplete hints even when I fully know the definitions 😂